run_dir/static/js/worksets.js

```diff
@@ -150,6 +150,33 @@ $(".tabbable").on("click", '[role="tab"]', function() {
       });
     });
   }
+  if($(this).attr('href')=='#tab_closed_worksets'){
+    $("#closed_ws_table_body").html('<tr><td colspan="4" class="text-muted"><span class="glyphicon glyphicon-refresh glyphicon-spin"></span> <em>Loading..</em></td></tr>');
+    return $.getJSON('/api/v1/closed_worksets', function(data) {
+      $("#closed_ws_table_body").empty();
+      $.each(data, function(key, value) {
+        if(!($.isEmptyObject(value))){
+          var tbl_row = $('<tr>');
+          tbl_row.append($('<td>').html(value['date_run']));
+          tbl_row.append($('<td>').html('<a href="/workset/'+key+'">'+'<font color="#000000">'+key+'</font>'+'</a>'));
```

Comment:
I think it's better not to modify the color here. The default color tells the user that its a link, which is not obvious if it's black. Maybe someone else objects?

Reply:
I just like it black as it is in the main worksets tab. Since I changed the color of the projects, it clashes with the default blue link color in my opinion. I think it clashes with black as well, but maybe Phil could have the final say in this ^^